src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/GetAzureADGroupMemberCommand.cs

```diff
@@ -38,6 +40,12 @@ public override void ExecuteCmdlet()
                 Paging = true
             };
 
+            PSADObject group = ActiveDirectoryClient.FilterGroups(options).FirstOrDefault();
+            if (group == null)
+            {
+                throw new KeyNotFoundException(string.Format(ProjectResources.GroupDoesntExist, GroupObjectId));
+            }
+
             do
             {
                 WriteObject(ActiveDirectoryClient.GetGroupMembers(options), true);
```

Comment:
@shuagarw Same comment:  if there is a change here, we need to document it for the release notes